recipes/nectarchain/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nectarchain" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nectarchain-{{ version }}.tar.gz
+  sha256: e421ecf3ef51f70cd0e033cdd03d2d638afb4f3886a361cfd79e874b5a5732f0
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python =3
+    - setuptools >=64.0.3
+    - setuptools-scm >=3.4
+    - pip
+  run:
+    - python =3
+    - ctapipe =0.12
+    - ctapipe-io-nectarcam
```

Comment:
Any reason this doesn't match upstream, which has:
```
    astropy~=4.2
    ctapipe~=0.12
    numpy~=1.22
    protozfits~=2.0
    tables>=3.7
```
?

Reply:
Ah, my mistake.

Reply:
The last commit (3eb328064acc251e89a8c28490fa3360116b805f) introduces the dependencies from upstream (e.g. on `astropy`, `ctapipe`, or `protozfits`), with light constraints as in #22372 , on which the presently proposed package depends.